### PR TITLE
Chore: fix small typos

### PR DESCRIPTION
**Pull request by dbmalkovsky**
There are also public methods with spelling mistakes that I didn't change (yet :smile: )

### CODE REVIEW

File: src/main/java/org/isf/telemetry/envdatacollector/constants/CollectorsConstants.java [hunk 1]
```diff
@@ -39,7 +39,7 @@ public interface CollectorsConstants {
 	String HW_CPU_NUM_PHYSICAL_PROCESSES = "hw_cpu_number_physical_processes";
 	String HW_CPU_NUM_LOGICAL_PROCESSES = "hw_cpu_number_logical_processes";
 	String HW_CPU_NAME = "hw_cpu_name";
-	String HW_CPU_IDENTIFIER = "hw_cpu_idientifier";
+	String HW_CPU_IDENTIFIER = "hw_cpu_identifier";
```
Comment (mwithi) [inline src/main/java/org/isf/telemetry/envdatacollector/constants/CollectorsConstants.java]: mmm I think this could be a problem when parsing on the remote server... could you please revert the change and open an issue? Thanks

Comment (dbmalkovsky) [reply src/main/java/org/isf/telemetry/envdatacollector/constants/CollectorsConstants.java]: Done

Comment (mwithi) [reply src/main/java/org/isf/telemetry/envdatacollector/constants/CollectorsConstants.java]: Thanks!

**Review by mwithi — APPROVED** [review]
LGTM. Thanks a lot!
